app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/profile/ProfilePresenterImpl.kt

```diff
@@ -19,12 +20,13 @@ class ProfilePresenterImpl(
 
     override fun requestProfileDetails(username: String) {
         view.showProgressBar(true)
-        provider.requestProfileDetails(username, object : PresenterCallback<Any> {
-            override fun onSuccess(o: Any?) {
+        provider.requestProfileDetails(username, object : PresenterCallback<ProfileDetailsResponse> {
+            override fun onSuccess(profileDetailsResponse: Any?) {
```

Comment:
You made the change, thanks. It looks a lot better, and cleaner.
I suggested to remove kotlin `Any` object because it's too "expansive" in build time. So, once you use `Generics`, you're already telling the proper object to generate for you.

I believe now, with your change, it's possible to replace the argument from `Any` to the `ProfileDetailsResponse` in your `overriden` function.
Is it possible?

Reply:
> Also, there are some files that you commited with your changes. Is it possible for you to remove it? I believe that can be done directly via github. Just deleting the files that are not necessary. Can you do that?

so i should delete the buildSrc folder?